Sources/PT.PM.Common/Nodes/Expressions/MultichildExpression.cs

```diff
@@ -38,6 +38,13 @@ public MultichildExpression()
         {
         }
 
+        public int GetDepth(int multichildExpressionArgNumber)
+        {
+            return 1 +
+                (Expressions.Count() > 2 && Expressions[multichildExpressionArgNumber] is MultichildExpression child ?
+                    child.GetDepth(multichildExpressionArgNumber) : 0);
+        }
```

Comment:
* What the meaning of this code? It's unclear. Can you please add a comment for explanation.
* `multichildExpressionArgNumber` is an external parameter. Maybe use for safe `Expressions.ElementAtOrDefault(multichildExpressionArgNumber)`?